### PR TITLE
Prescription patient account and profile form steps [WEB-819,WEB-820]

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
See [WEB-819] and [WEB-820] for details.

Includes the prescription form with the patient account and profile steps completed, and the others seeded with placeholder content.  The routes currently only require authentication.  The authorization constraints are still TBD, and will be implemented at a later stage

This also includes a roughly-scaffolded prescriptions table and associated route, which is not intended to be fully reviewed at this point (though happy to include any obvious opportunities to improve it if anything comes up).  The prescriptions table, at this point, is primarily for debugging and navigating to the various saved prescriptions.

On the note of saved prescriptions, they are being temporarily stored in localStorage under the `prescriptions` key.  They will be moved to using the near-ready backend prescription service at a later time.  Prescriptions should be updated in the `prescriptions` store at the end of each main form step.

As well as saving to localStorage at the end of each form step, the state of the form is continuously persisted to localStorage via the `formik-persist` library.  The storage is cleared whenever a new 'Add Prescription' process is started, or when an existing form is being edited, in which case it is then replaced by the stored values.

There are also a few shuffles as expected to some of the lower-level components we've been working on, in order to improve the ease of theming them.

The unit tests are in place, with the following caveats:
1. I didn't write any tests for the `Prescriptions` table view, as that feature is only part of this PR for primarily debugging reasons at this point, and will be completed and tested as part of a separate PR
2. The `PrescriptionForm` tests will need to be added to once we wire in the backend service and have proper handlers in place.
3. The `useLocalStorage` hook is not tested, as it's only purpose is to mock the missing backend prescription store, and will likely go away when it's no longer needed.

[WEB-819]: https://tidepool.atlassian.net/browse/WEB-819
[WEB-820]: https://tidepool.atlassian.net/browse/WEB-820

### CODE REVIEW

File: app/pages/prescription/profileFormSteps.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+              id="phoneNumber.number"
+              label={t('Patient Phone Number')}
+              error={getFieldError(meta.phoneNumber.number)}
+              {...inputStyles}
+            />
+          </InputMask>
+        )}
+      />
+      <Text fontSize={0}>
+        {t('The patient\'s phone number may be used to provided direct assistance regarding their Tidepool account. Standard messaging rates may apply.')}
```
Comment (krystophv) [inline app/pages/prescription/profileFormSteps.js:55]:
```suggestion
        {t('The patient\'s phone number may be used to provide direct assistance regarding their Tidepool account. Standard messaging rates may apply.')}
```

File: test/unit/app/core/forms.test.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  const touchedAndError = {
+    touched: true,
+    error: 'error!',
+  };
+
+  const touchedAndNoError = {
+    touched: true,
+    error: null,
+  };
+
```
Comment (krystophv) [inline test/unit/app/core/forms.test.js:22]:
```suggestion

```

File: app/components/elements/Stepper.js [hunk 3]
```diff
@@ -67,18 +74,37 @@ export const Stepper = props => {
...
+  };
+
+  setInitialActiveStepStateFromParams();
+
+  const [transitioningToStep, setTransitioningToStep] = React.useState();
   const [activeStep, setActiveStep] = React.useState(initialActiveStepState);
   const [activeSubStep, setActiveSubStep] = React.useState(initialActiveSubStepState);
+  const prevActiveStep = usePrevious(activeStep);
+
+  window.top.addEventListener('popstate', (e) => {
```
[the remark this answers is not in the document]
Comment (cbwebdevelopment) [reply app/components/elements/Stepper.js:97]: Good idea 👍  
I'll implement this and ping you when ready.

**Comment by cbwebdevelopment** [comment]
Moved the window event listener handling to a `useEffect` hook in 1713d50.

Ready for review

**Pull request merged**